src/scheduler/pbs_bitmap.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 1994-2018 Altair Engineering, Inc.
+ * For more information, contact Altair at www.altair.com.
+ *
+ * This file is part of the PBS Professional ("PBS Pro") software.
+ *
+ * Open Source License Information:
+ *
+ * PBS Pro is free software. You can redistribute it and/or modify it under the
+ * terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or (at your option) any
+ * later version.
+ *
+ * PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Commercial License Information:
+ *
+ * For a copy of the commercial license terms and conditions,
+ * go to: (http://www.pbspro.com/UserArea/agreement.html)
+ * or contact the Altair Legal Department.
+ *
+ * Altair’s dual-license business model allows companies, individuals, and
+ * organizations to create proprietary derivative works of PBS Pro and
+ * distribute them - whether embedded or bundled with other software -
+ * under a commercial license agreement.
+ *
+ * Use of Altair’s trademarks, including but not limited to "PBS™",
+ * "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+ * trademark licensing policies.
+ *
+ */
+
+#ifndef _PBS_BITMASK_H
+#define _PBS_BITMASK_H
+#ifdef	__cplusplus
+extern "C" {
+#endif
+
+
+struct pbs_bitmap {
+	unsigned long *bits;	/* bit storage */
+	long num_longs;		/* number of longs in the bits array */
```

Comment:
I'd rather leave it alone.  The data inside the bitmap are the bits themselves.  It just so happens we store them in longs.  If we called it num_elems or total_count, it could easily get confused that this is the number of bits, not longs.  There is a comment which explains what it is. Is that enough?